### PR TITLE
Reducing code duplication by moving duplicated logic from try and cat…

### PR DESCRIPTION
**Pull request by DawidMazurek**
…ch to single finally block

#244

### CODE REVIEW

File: src/Broadway/EventHandling/SimpleEventBus.php [hunk 1]
```diff
@@ -49,12 +49,12 @@ public function publish(DomainEventStreamInterface $domainMessages)
                         $eventListener->handle($domainMessage);
                     }
                 }
-
-                $this->isPublishing = false;
             } catch (Exception $e) {
-                $this->isPublishing = false;
                 throw $e;
             }
+            finally {
```
Comment (asm89) [inline src/Broadway/EventHandling/SimpleEventBus.php]: nit: `} finally {`

**Comment by asm89** [comment]
lgtm :+1:

**Comment by GromNaN** [comment]
The catch part is not necessary.

**Comment by wjzijderveld** [comment]
Thanks @DawidMazurek

**Pull request merged**